cpp/data/installer/scripts/install_ubtools.sh

```diff
@@ -5,6 +5,21 @@
 # compile cpp installer
 # run it
 
+# ./install_ubtools.sh parameter
```

Comment:
These additional CLI options should not be documented here, because they will just be passed through to the installer. This is documented in the wiki. If you want to add a comment here, please just add a comment that refers to either the wiki page or installer.cc.